docs/self-managed/operational-guides/backup-restore/_harmonized-indexes-main.md

```diff
@@ -0,0 +1,232 @@
+```mermaid
+erDiagram
+
+    camunda-authorization {
+        keyword id
+        long ownerKey
+        keyword ownerType
+        keyword resourceType
```

Comment:
❓ One thing I was wondering was whether we could turn the values around. That we first have the name of the property then the type? I felt this is somehow more natural. What are your thoughts? 

For example this is also done in the C7 ER diagram https://docs.camunda.org/manual/7.22/user-guide/process-engine/database/database-schema/#entity-relationship-diagrams